### PR TITLE
[Merged by Bors] - chore: remove some unused variables

### PR DESCRIPTION
**Pull request by adomani**
Inspired by #17178 and found by a linter.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by kim-em** [comment]
bors d+

**Comment by adomani** [comment]
!bench

**Comment by adomani** [comment]
bors merge

File: Mathlib/Algebra/Field/Basic.lean [hunk 1]
```diff
@@ -167,7 +167,7 @@ end DivisionRing
 
 section Semifield
 
-variable [Semifield K] {a b c d : K}
+variable [Semifield K] {a b d : K}
```
[the remark this answers is not in the document]
Comment (adomani) [reply Mathlib/Algebra/Field/Basic.lean:170]: I see your point, but you can see that `c` and `d` alternate being implicit/explicit/non-present in the following lemmas and I do not think that `d` is ever used *before* `c`!

**Pull request closed without merging**